crates/node/service/src/actors/network/mod.rs

```diff
@@ -0,0 +1,13 @@
+//! Network Actor
+
+mod actor;
+mod builder;
+mod driver;
+mod error;
+mod handler;
+
+pub use actor::{NetworkActor, NetworkActorError, NetworkContext, NetworkInboundData};
+pub use builder::NetworkBuilder;
+pub use driver::{NetworkDriver, NetworkDriverError};
+pub use error::NetworkBuilderError;
+pub use handler::NetworkHandler;
```

Comment:
Can we re-arrange this so each module is above its public `use` to keep it consistent with the rest of kona?